Units/simple-robot.d/input.robot

```diff
@@ -18,24 +18,23 @@ Test case 1
     Keyword 3  @{list 2}
     Keyword 3  @{list_2}
     Keyword with a variable name
-    Keyword two with a variable name
     it's ok to be correct
 
-*** Keyword ***
+*** Keywords ***
 Keyword 1
+    [Arguments]  ${arg}
     python keyword 1
 
 Keyword 2
+    [Arguments]  ${arg}
     python_keyword_2
 
 Keyword_3
+    [Arguments]  @{arg1}
```

Comment:
what happens if `[Arguments]` is placed at the same line of the keyword like:

`Keyword_4 [Arguemtns] ...`

As far as I know this is valid syntax. In this case ` [Arguemtns] ...` should not be in part of a tag name.